src/schema/v2/conversation/message.ts

```diff
@@ -134,6 +134,13 @@ export const MessageType = new GraphQLObjectType<any, ResolverContext>({
       resolve: ({ metadata }) => isInvoiceMessage(metadata),
     },
     createdAt: date(),
+    sentAt: date(),
+    messageSentOnPlatform: {
```

Comment:
#minor could possibly follow the `is-` prefix for booleans

Reply:
Fixed 👍